gammapy/data/tests/test_observationlist.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_data
+from ...utils.energy import EnergyBounds
+from ...data import DataStore, ObservationList
+
+
+@requires_data('gammapy-extra')
+def test_make_psftable():
```

Comment:
ok!!
I don't really have an example. But if you take the code in test_observationlist.py and try to access for examples psf1.psf_value or psf2.psf_value, you will see that there are completely different!!  


```
sig1=Angle(np.array([ 0.02858012,  0.1211779 ,  0.0515614 ]), 'deg')
norm1=np.array([ 0.1735659 ,  0.41401832,  0.41241578])
 sig2=Angle(np.array([ 0.02559773,  0.10916024,  0.04580572]), 'deg')
 norm2=np.array([ 0.17220046,  0.40929013,  0.41850941])
```